documentation/creating-a-new-stack.md

```diff
@@ -0,0 +1,17 @@
+# Prerequisites
+* Follow the [Setup Environment](setup-environment.md) wiki to setup your development environment.
+* Follow the [Configuring a Stack](configuring-a-stack.md) wiki to configure your development stack settings.
+* Follow the [Releasing](releasing.md) wiki to create a development release on the bucket's created in step 1.
+* Log in using these [instructions](aws-account.md#aws-command-line)
+* Connect to your local container using [rake app:sh](local-environment.md)
+
+# Creating a New Stack
+1. Verify your `.env` file is pointed to the correct Givesource stack. In this case it will be one that does not yet exist on AWS.
+2. Verify the latest version is available in the release buckets: see [Releasing](releasing.md).
+3. Create the stack:
+	```
+	npm run cloudformation:create
```

Comment:
Anymore I've been doing:

```
rake aws:login
rake app:build <-- if not built
rake app:sh
npm run cloudformation:create

Reply:
The login and build are both referenced in the `Prerequisites` section. I'm fine expanding that or making it more explicit if we want to - but I was trying to avoid a complete re-write of this documentation